pkg/ccl/streamingccl/streamingest/stream_ingestion_dist.go

```diff
@@ -56,6 +55,10 @@ var replanFrequency = settings.RegisterDurationSetting(
 	settings.PositiveDuration,
 )
 
+// replicationPartitionInfoFilename is the filename at which the replication job
+// resumer writes its partition specs.
+const replicationPartitionInfoFilename = "~replication-partition-specs.binpb"
```

Comment:
nit: is this used? if not, sad the linters did not catch this...

Reply:
it is! in `persistStreamIngestionPartitionSpecs` and the read path when generating the file. Its where we read/write the replication partition specs.